localnet/single/README.md

```diff
@@ -20,19 +20,33 @@
 ./stop.sh
 ```
 
+## Run with docker-compose
+```shell
+# Start the finschia node and explorer and faucet
+docker-compose up
+
+# Stop 
+docker-compose down
```

Comment:
I think it is better to showed options `-d or -v` like README in with-ibc
https://github.com/Finschia/hackathon/blob/main/localnet/with-ibc/README.md?plain=1#L12-L18